modules/telemetry/v1/data_block.py

```diff
@@ -128,116 +181,127 @@ def __str__(self):
         return f"{self.__class__.__name__} -> time: {self.mission_time} ms, altitude: {self.altitude} m"
 
     def __iter__(self):
-        yield "mission time", self.mission_time
-        yield "altitude", {"meters": self.altitude}
-
-
-class PressureDB(DataBlock):
-    """Represents a pressure data block."""
+        yield "mission_time", self.mission_time
+        yield "altitude", {"metres": self.altitude, "feet": metres_to_feet(self.altitude)}
 
-    def __init__(self, mission_time: int, pressure: int) -> None:
-        """
-        Constructs a pressure data block.
 
-        Args:
-            mission_time: The mission time the altitude was measured at in milliseconds since launch.
-            pressure: The pressure in millibars.
+class TemperatureDB(DataBlock):
+    """Represents a temperature data block."""
 
-        """
+    def __init__(self, mission_time: int, temperature: int) -> None:
         super().__init__(mission_time)
-        self.pressure: int = pressure
+        self.temperature: int = temperature
 
     @classmethod
     def from_bytes(cls, payload: bytes) -> Self:
         """
-        Constructs a pressure data block from bytes.
+        Constructs a temperature data block from bytes.
         Returns:
-            A pressure data block.
+            A temperature data block.
         """
-        parts = struct.unpack("<II", payload)
+        parts = struct.unpack("<Ii", payload)
         return cls(parts[0], parts[1])
 
     def __len__(self) -> int:
         """
-        Get the length of a pressure data block in bytes.
+        Get the length of a temperature data block in bytes.
         Returns:
-            The length of a pressure data block in bytes, not including the block header.
+            The length of a temperature data block in bytes, not including the block header.
         """
         return 8
 
     def __str__(self):
-        return f"{self.__class__.__name__} -> time: {self.mission_time} ms, pressure: {self.pressure} Pa"
+        return (
+            f"{self.__class__.__name__} -> time: {self.mission_time} ms, "
+            f"temperature: {self.temperature} mC "
+            f"({round(self.temperature / 1000, 1)}°C)"
+        )
 
     def __iter__(self):
         yield "mission_time", self.mission_time
-        yield "pressure", {"pascals": self.pressure}
+        yield "temperature", {"millidegrees": self.temperature, "celsius": round(self.temperature / 1000, 2)}
```

Comment:
Nit: Add converter function from millidegrees to celsius since we have one for meters to feet